lib/containers/urls.pm

```diff
@@ -260,6 +262,13 @@ sub supports_image_arch {
     (grep { $_ eq $arch } @{$images_uri{$distri}{$version}{available_arch}}) ? 1 : 0;
 }
 
+# Returns a tuple of image urls.
+# If empty, no images available.
+sub get_urls_from_var {
```

Comment:
This name is too unspecific. Or rename `get_urls_from_var` to something more specific e.g. `get_container_image_urls` or pass the var as argument, e.g.

```suggestion
# Return tuple of image urls from the specified variable
sub get_urls_from_var {
    my $var = $_[0];
    my @urls = split(/,/, trim(get_var($var, "")));
    s{^\s+|\s+$}{}g foreach @urls;         # Trim each element in the array
    return (\@urls);
}
```

Reply:
I don't agree. There is already `get_suse_container_urls` so I find `get_urls_from_var` as the specific option.

Reply:
I added the comment. Please check again.